backends-common/rabbitmq/src/test/java/org/apache/james/backend/mailqueue/DockerRabbitMQ.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations      *
  * under the License.                                           *
  ****************************************************************/
-package org.apache.james.queue.rabbitmq;
+package org.apache.james.backend.mailqueue;
```

Comment:
I don't agree with the package name : it should be org.apache.james.backend.rabbitmq